test/value_parser_test.rb

```diff
@@ -85,17 +85,18 @@ class FilterTest < ActiveSupport::TestCase
     assert_equal result.max, end_time.to_s
   end
 
-  test "parses range for Date string range" do
+  test "parses range for Date string range and normalizes Date values" do
     options = {
       supports_ranges: true
     }
-    start_date = Date.new(2018, 1, 26)
-    end_date = Time.new(2018, 1, 29)
+
+    start_date = "2018-01-01T10:00:00Z[Etc/UTC]"
+    end_date = "2018-01-01T12:00:00Z[Etc/UTC]"
     range_string = "#{start_date}...#{end_date}"
-    parser = Sift::ValueParser.new(value: range_string, options: options)
+    parser = Sift::ValueParser.new(value: range_string, type: :datetime, options: options)
```

Comment:
should we also add a test for `:time` and `:date` types?

Reply:
added tests